equational_theories/Basic.lean

```diff
@@ -188,6 +188,18 @@ theorem Equation43_not_implies_Equation42 : ∃ (G: Type) (_: Magma G), Equation
   dsimp [hG] at h
   linarith
 
+theorem Equation43_not_implies_Equation387 : ∃ (G: Type) (_: Magma G), Equation43 G ∧ ¬ Equation387 G := by
+  let hG : Magma Nat := {
+    op := fun x y => x+y
+  }
+  use ℕ, hG
+  constructor
+  . exact Nat.add_comm
+  by_contra h
+  replace h := h 0 1
```

Comment:
You can use `specialize h 0 1` here

Reply:
Looks like elsewhere in the file we used the form `replace h := h ...` too. Shall I replace all of them?

Reply:
Sure, we can do that in a separate PR

Reply:
I've gone ahead and implemented the change globally.